griptape/tasks/prompt_task.py

```diff
@@ -28,6 +31,9 @@ class PromptTask(RuleMixin, BaseTask):
         default=Factory(lambda self: self.default_generate_system_template, takes_self=True),
         kw_only=True,
     )
+    conversation_memory: Union[Optional[BaseConversationMemory], Literal[NOTHING]] = field(  # pyright: ignore[reportInvalidTypeForm] TODO: Replace with [NothingType](https://github.com/python-attrs/attrs/pull/1358)
+        default=Factory(lambda: NOTHING), kw_only=True
+    )
```

Comment:
we can steal the `TypeAlias` implementation from that PR if that would make pyright happier

Reply:
I don't see how their PR is making pyright happy since it still uses `Literal[Nothing]`.